pkg/token/options_test.go

```diff
@@ -149,9 +166,53 @@ func TestOptionsWithEnvVars(t *testing.T) {
 			}
 			o.AddFlags(&pflag.FlagSet{})
 			o.UpdateFromEnv()
-			if o != tc.expected {
+			if !cmp.Equal(o, tc.expected, cmp.AllowUnexported(Options{})) {
 				t.Fatalf("expected option: %+v, got %+v", tc.expected, o)
 			}
 		})
 	}
 }
+
+func TestParsePoPClaims(t *testing.T) {
+	t.Run("pop-claim parsing should fail on empty string", func(t *testing.T) {
+		popClaims := ""
+		if _, err := parsePopClaims(popClaims); err == nil || !strings.Contains(err.Error(), "no claims provided") {
+			t.Fatalf("parsing pop claims should return error if claims is an empty string. got: %s", err)
+		}
+	})
+
+	t.Run("pop-claim parsing should fail on whitespace-only string", func(t *testing.T) {
+		popClaims := "	    "
+		if _, err := parsePopClaims(popClaims); err == nil || !strings.Contains(err.Error(), "no claims provided") {
+			t.Fatalf("parsing pop claims should return error if claims is whitespace-only. got: %s", err)
+		}
+	})
+
+	t.Run("pop-claim parsing should fail if claims are not provided in key=value format", func(t *testing.T) {
+		popClaims := "claim1=val1,claim2"
+		if _, err := parsePopClaims(popClaims); err == nil || !strings.Contains(err.Error(), "Ensure the claims are formatted as `key=value`") {
+			t.Fatalf("parsing pop claims should return error if claims are not provided in key=value format. got: %s", err)
+		}
+	})
+
+	t.Run("pop-claim parsing should fail if claims are malformed", func(t *testing.T) {
+		popClaims := "claim1=  "
+		if _, err := parsePopClaims(popClaims); err == nil || !strings.Contains(err.Error(), "Ensure the claims are formatted as `key=value`") {
+			t.Fatalf("parsing pop claims should return error if claims are malformed. got: %s", err)
+		}
+	})
+
+	t.Run("pop-claim parsing should fail if u-claim is not provided", func(t *testing.T) {
+		popClaims := "claim1=val1, claim2=val2"
```

Comment:
`"claim1=val1, claim2=val2"` CLI will parse this option without the space right ? `"claim1=val1,claim2=val2"`

Reply:
Yeah the pop token claim parsing trims spaces from each key/value